pkg/emscripten/libretro-thread/libretro.worker.js

```diff
@@ -1,66 +1,400 @@
-importScripts("zip-no-worker.min.js");
-
-async function writeFile(path, data) {
-  const root = await navigator.storage.getDirectory();
-  const dir_end = path.lastIndexOf("/");
-  const parent = path.substr(0, dir_end);
-  const child = path.substr(dir_end+1);
-  const parent_dir = await mkdirTree(parent);
-  const file = await parent_dir.getFileHandle(child,{create:true});
-  const stream = await file.createSyncAccessHandle();
-  const written = stream.write(data);
-  stream.close();
-}
-
-async function mkdirTree(path) {
-  const root = await navigator.storage.getDirectory();
-  const parts = path.split("/");
-  let here = root;
-  for (const part of parts) {
-    if (part == "") { continue; }
-    here = await here.getDirectoryHandle(part, {create:true});
-  }
-  return here;
+let root, BFS, BFSDB;
+let bundleCounter = 0;
+let loadedScripts = [];
+const FS = {};
+const helper = {};
+
+// this is huge and takes between 2 and 3 minutes to unzip. (10 minutes for firefox?)
+// luckily it only needs to be done once.
+const bundlePath = ["assets/frontend/bundle.zip.aa",
```

Comment:
Yeah, it's a shame CORS stuff got in the way of using the smaller bundle from the buildbot.  This is a priority to fix, we should get in touch with @m4xw or whoever is in charge of the buildbot's http headers.